### PR TITLE
Fix `check-redirects` workflow error

### PR DESCRIPTION
**Pull request by cw118**
#### Summary/Motivation

Politely poking @mdn/core-yari-dev (and @caugner if I may) for a review :)

Following mdn/yari#6386, `yari/tool/cli.js` was converted to TypeScript and is now found at `yari/tool/cli.ts`, which I believe is causing the Check Redirects workflow to run with errors.

This PR updates `package.json` so that `yarn content` uses the path `yari/tool/cli.ts` (with the correct `.ts` file ending), to hopefully fix the [workflow errors](https://github.com/mdn/content/actions/workflows/pr-check_redirects.yml). No other workflow seems to be affected.

#### Supporting details

The workflow error:

```
$ env-cmd --silent cross-env CONTENT_ROOT=files node node_modules/@mdn/yari/tool/cli.js validate-redirects en-us --strict
node:internal/modules/cjs/loader:93[6](https://github.com/mdn/content/runs/6712996419?check_suite_focus=true#step:5:7)
  throw err;
  ^

Error: Cannot find module '/home/runner/work/content/content/node_modules/@mdn/yari/tool/cli.js'
```

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "up-to-date-check": "node scripts/up-to-date-check.js",
     "start": "yarn up-to-date-check && env-cmd --silent cross-env CONTENT_ROOT=files REACT_APP_DISABLE_AUTH=true BUILD_OUT_ROOT=build node node_modules/@mdn/yari/server",
     "filecheck": "env-cmd --silent cross-env CONTENT_ROOT=files node node_modules/@mdn/yari/filecheck/cli.js --cwd=.",
-    "content": "env-cmd --silent cross-env CONTENT_ROOT=files node node_modules/@mdn/yari/tool/cli.js",
+    "content": "env-cmd --silent cross-env CONTENT_ROOT=files node node_modules/@mdn/yari/tool/cli.ts",
```
Comment (schalkneethling) [inline package.json:13]:
```suggestion
    "content": "env-cmd --silent cross-env CONTENT_ROOT=files ts-node node_modules/@mdn/yari/tool/cli.ts",
```

**Comment by cw118** [comment]
Thank you so much for the quick review @schalkneethling ! I have committed the suggestion :)

File: package.json (hunk 1 above)
Comment (cw118) [reply package.json]: @caugner Should I convert this to a draft for now?

**Comment by caugner** [comment]
@cw118 I have fast-tracked and self-approved https://github.com/mdn/content/pull/16911 to fix the failing scripts, and will provide a fix for yari in https://github.com/mdn/yari/pull/6457.

**Comment by cw118** [comment]
> @cw118 I have fast-tracked and self-approved #16911 to fix the failing scripts, and will provide a fix for yari in [mdn/yari#6457](https://github.com/mdn/yari/pull/6457). We shouldn't have to add/use `ts-node` in mdn/content.

That's great, I will close this then :) Thank you so much!

**Pull request closed without merging**